plugins/AccountStats.py

```diff
@@ -78,9 +82,9 @@ def get_db_version(self):
         return self.db.execute("PRAGMA user_version").fetchone()[0]
 
     def fetch_history(self, first_time_stamp, last_time_stamp):
-        history = self.api.return_lending_history(first_time_stamp, last_time_stamp - 1, 50000)
+        history = self.api.return_lending_history(first_time_stamp, last_time_stamp - 1, 5000)
```

Comment:
pared this down as to query more frequently with less chance of large data bursts getting blocked or not fully sent by exchange